app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -79,6 +79,29 @@ def metrics_connect
                                      authentication_password('default'))
     end
 
+    def self.raw_operations_connect(hostname, port, username, password)
+      require 'hawkular_all'
+      host_port = URI::HTTP.build(:host => hostname, :port => port.to_i).to_s
+      host_port.sub!('http://', '') # Api can't internally deal with the schema
+      credentials = {:username => username, :password => password}
+      ::Hawkular::Operations::OperationsClient.new(:host => host_port, :credentials => credentials)
+    end
+
+    def operations_connect
+      self.class.raw_operations_connect(hostname,
+                                        port,
+                                        authentication_userid('default'),
+                                        authentication_password('default'))
+    end
+
+    def reload_middleware_server(path)
```

Comment:
Is `path` always going to be the `ems_ref`?  If so, I think `ems_ref` is a better parameter name here, or even `provider_object_id`.   (same with `stop_middleware_server`)